token-metadata/program/Cargo.toml

```diff
@@ -36,6 +36,7 @@ solana-sdk = "1.14"
 solana-program-test = "1.14"
 serde = { version = "1.0.147", features = ["derive"]}
 async-trait = "0.1.64"
+old-token-metadata = { package = "mpl-token-metadata", version = "=1.10.0", features = ["no-entrypoint"] }
```

Comment:
I see you're importing this to test backwards compatibility which is very clever. Just curious to know what the process with these "temporary" tests going forward? For instance, say we need to do something similar in a future PR, would we have to import `old-token-metadata` and `old-token-metadata-for-the-new-pr` to ensure they each have the required version? Or are we planning on cleaning up that test once merge?

Reply:
I was going to remove the test up soon in a follow on PR.  I think we will have to remove them as cargo does not like multiple versions of a dependency.   For example, when I try adding a new `old-token-metadata-for-the-new-pr`, I get a "failed to select version" error:

```
    error: failed to select a version for `mpl-token-metadata`.
        ... required by package `mpl-token-metadata v1.10.0 (/home/danenbm/Metaplex-Workspace/metaplex-program-library/token-metadata/program)`
    versions that meet the requirements `=1.9.1` are: 1.9.1
    
    all possible versions conflict with previously selected packages.
    
      previously selected package `mpl-token-metadata v1.10.0`
        ... which satisfies dependency `old-token-metadata = "=1.10.0"` of package `mpl-token-metadata v1.10.0 (/home/danenbm/Metaplex-Workspace/metaplex-program-library/token-metadata/program)`
    
    failed to select a version for `mpl-token-metadata` which could resolve this conflict
```